### PR TITLE
string/words

### LINKED ISSUE #190: string/words

### PR DESCRIPTION
**Pull request by bookercodes**
Closes #190.

### CODE REVIEW

File: string/words.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+Extracts words from str.
+@module string/words
+@param {string} str The string from which to extract the words.
+@param {pattern} [pattern=] The pattern to match the words.
```
Comment (bookercodes) [inline string/words.js]: I'm not too sure how to communicate that the parameter is optional. Please advise.

Comment (EvanHahn) [inline string/words.js]: `[pattern=/\w+/g]`

Comment (EvanHahn) [inline string/words.js]: `{pattern}` should be `{regexp}` or whatever JSDoc uses for regular expressions.

Comment (bookercodes) [inline string/words.js]: Ta.

**Comment by bookercodes** [comment]
Updated the docs.

**Comment by EvanHahn** [comment]
Perfect! Thanks.

**Pull request merged**